src/test/java/selenium/webdriver/SeleniumEasyTest.java

```diff
@@ -0,0 +1,52 @@
+package selenium.webdriver;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.support.ui.Select;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class SeleniumEasyTest {
+    private WebDriver driver;
+
+    @BeforeEach
+    public void setup() {
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--ignore-ssl-errors=yes", "--ignore-certificate-errors");
+        driver = new ChromeDriver(options);
+    }
+
+    @Test
+    @DisplayName("Test with multiselect")
+    public void multiSelect() throws InterruptedException {
+        driver.get("https://demo.seleniumeasy.com/basic-select-dropdown-demo.html");
+        WebElement selectElement = driver.findElement(By.id("multi-select"));
+        Select select = new Select(selectElement);
+        select.selectByIndex(1);
+        select.selectByIndex(3);
+        select.selectByIndex(5);
+        List<WebElement> selectedOptions = select.getAllSelectedOptions();
+        List<String> names = new ArrayList<>();
+        for (WebElement selectedOption : selectedOptions) {
+            String text = selectedOption.getText();
+            names.add(text);
+        }
+        assertEquals(3, names.size(), "Wrong number of selected elements");
+        assertEquals(List.of("Florida", "New York", "Texas"), names, "Wrong items are selected");
```

Comment:
Just imagine if the order in the list of states will be changed by someone, than your test fail because you hardcoded the names here. Let's compare the names dynamically

Reply:
Done